src/keyboard.re

```diff
@@ -0,0 +1,57 @@
+type subscription;
+
+type event('a) =
+  {
+    ..
+    "endCoordinates": {
+      .
+      "height": float,
+      "screenX": float,
+      "screenY": float,
+      "width": float
+    }
+  } as 'a;
+
+type listener('a) = event('a) => unit;
+
+type keyboardEvent = [
```

Comment:
We are migrating away from polymorphic variants. Please change it to a normal variants. Long story short it makes error messages much better.